docs/datatypes/complex_numbers.md

```diff
@@ -87,24 +88,22 @@ Test whether a complex number equals another complex value.
 
   Two complex numbers are equal when both their real and imaginary parts are
   equal.
-  
-### complex.format([precision: number])
 
-Get a string representation of the complex number,
-  formatted as `a + bi` where `a` is the real part and `b` the imaginary part.
-  If precision is defined, the units value will be rounded to the provided
-  number of digits.
-  
-### complex.fromJSON(json)
+### complex.neg()
 
-Revive a complex number from a JSON object. Accepts
-  An object `{mathjs: 'Complex', re: number, im: number}`, where the property
-  `mathjs` is optional.
-  Used when deserializing a complex number, see [Serialization](../core/serialization.md).
+Returns a complex number with a real part and an imaginary part equal in magnitude but opposite in sign.
```

Comment:
nit: adding "to the current complex number" to the end of this sentance would improve clariry.

Reply:
done :)